pkg/selinux/selinux-client.js

```diff
@@ -129,11 +141,18 @@ export function init(statusChangedCallback) {
                     output = output.split("~~~~~");
                     status.shell = output[0];
                     status.modifications = [];
+                    status.ansible = "";
+
+                    for (let i = 1; i < output.length; i++) {
```

Comment:
`output.forEach((line, i) => {...}`

Reply:
It skips the first item (starting from i=1). I find it a but cleaner than doing either `if (i < 1) return;` or `slice(1)`.

Reply:
Sorry, I missed the "1".